mixer/adapter/stackdriver/metric/bufferedClient.go

```diff
@@ -166,7 +167,12 @@ func (b *buffered) Send() {
 		if err != nil {
 			ets := handleError(err, timeSeries)
 			b.updateRetryBuffer(ets)
-			_ = b.l.Errorf("Stackdriver returned: %v\nGiven data: %v", err, timeSeries)
+			b.l.Errorf("%d time series was sent and Stackdriver returned: %v\n", len(timeSeries), err) // nolint: errcheck
+			if isOutOfOrderError(err) {
```

Comment:
should we record some sort of metric here that we can monitor?

Reply:
Yeah, I can do it in follow up PR.